### PR TITLE
feat: sdk-2020-04-13 release 

### PR DESCRIPTION
**Pull request by germanattanasio**
This PR contains the changes for the v8.4.0 release of the Java SDK. 

The tags used for this release are:
```
API definitions: sdk-2020-04-13
Generator: 2.2.5
```

### CODE REVIEW

File: assistant/src/main/java/com/ibm/watson/assistant/v1/model/Context.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2017, 2020.
+ * (C) Copyright IBM Corp. 2020.
```
Comment (mamoonraja) [inline assistant/src/main/java/com/ibm/watson/assistant/v1/model/Context.java:2]: good old copy right issue

File: assistant/src/main/java/com/ibm/watson/assistant/v1/model/CounterexampleCollection.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2017, 2020.
+ * (C) Copyright IBM Corp. 2020.
```
Comment (mamoonraja) [inline assistant/src/main/java/com/ibm/watson/assistant/v1/model/CounterexampleCollection.java:2]: again

File: speech-to-text/src/main/java/com/ibm/watson/speech_to_text/v1/SpeechToText.java [hunk 3]
```diff
@@ -425,10 +448,12 @@ public WebSocket recognizeUsingWebSocket(RecognizeOptions recognizeOptions, Reco
...
+      urlBuilder.addQueryParameter(
+          "language_customization_id", recognizeOptions.languageCustomizationId());
     }
     if (recognizeOptions.acousticCustomizationId() != null) {
-      urlBuilder.addQueryParameter("acoustic_customization_id", recognizeOptions.acousticCustomizationId());
+      urlBuilder.addQueryParameter(
+          "acoustic_customization_id", recognizeOptions.acousticCustomizationId());
     }
     if (recognizeOptions.baseModelVersion() != null) {
       urlBuilder.addQueryParameter("base_model_version", recognizeOptions.baseModelVersion());
```
Comment (mamoonraja) [inline speech-to-text/src/main/java/com/ibm/watson/speech_to_text/v1/SpeechToText.java:459]: I think you need to add `speech_detector_sensitivity` and `background_audio_suppression` to `urlbuilder`

Comment (germanattanasio) [reply speech-to-text/src/main/java/com/ibm/watson/speech_to_text/v1/SpeechToText.java:459]: @mamoonraja are those parameters part of the query to the service during a websockey or part of the start message in the websocket?

File: assistant/src/main/java/com/ibm/watson/assistant/v2/model/MessageContextSkill.java [hunk 4]
```diff
@@ -12,37 +12,30 @@
...
-  /**
-   * Builder.
-   */
+  protected MessageContextSkillSystem system;
+
+  /** Builder. */
   public static class Builder {
     private Map<String, Object> userDefined;
-    private Map<String, Object> system;
+    private MessageContextSkillSystem system;
```
Comment (mediumTaj) [inline assistant/src/main/java/com/ibm/watson/assistant/v2/model/MessageContextSkill.java:30]: We should look to see if this is breaking

**Pull request closed without merging**